ncbitax2lin/README.md

```diff
@@ -0,0 +1,89 @@
+# NCBItax2lin
```

Comment:
should we put all the code here?  could you just put ncbitax2lin into Dockerfile? and maybe write a quick script to run the lineage?

Reply:
ok, I forked ncbitax2lin and made it a separate repo that gets cloned inside the rake task